common/primitives/src/offchain.rs

```diff
@@ -22,9 +22,10 @@ sp_externalities::decl_extension! {
 }
 
 /// runtime new customized
+#[cfg(not(feature = "no-host-functions"))]
```

Comment:
We need other host functions I think to function
```suggestion
#[cfg(not(feature = "no-custom-host-functions"))]
```